pages/studentAssessmentHomework/studentAssessmentHomework.sql

```diff
@@ -46,42 +46,39 @@ WITH log AS (
 )
 INSERT INTO group_logs
     (authn_user_id, user_id, group_id, action)
-SELECT $user_id, $user_id, group_id, 'join'
+SELECT $authn_user_id, $user_id, group_id, 'join'
 FROM log;
 
 
 -- BLOCK create_group
-WITH log AS (
+WITH
+create_group AS (
     INSERT INTO groups
         (name, group_config_id, course_instance_id)
-    VALUES
-        (
-            $group_name,
-            (SELECT id FROM group_configs WHERE assessment_id = $assessment_id AND deleted_at IS NULL),
-            (SELECT course_instance_id FROM group_configs WHERE assessment_id = $assessment_id AND deleted_at IS NULL)
-        )
+    (
+        SELECT 
+            $group_name, gc.id, gc.course_instance_id
+        FROM 
+            group_configs AS gc
+        WHERE
+            gc.assessment_id = $assessment_id
+            AND deleted_at IS NULL
```

Comment:
```suggestion
            AND gc.deleted_at IS NULL
```

Reply:
@frankyhao can you check that this change is ok?

Reply:
Yeah, I just manually check the functionality of creating groups.
Sorry, I forgot to add "gc." before deleted_at. The previous version could still pass CI because we only have 1 table in FROM, not ambiguous. It is definitely better to have a "gc." to specific table and decrease risks in future development.

Reply:
Thanks!